ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -8,13 +8,13 @@ import {ClrIcon, InfoIcon} from 'app/components/icons';
 import {TooltipTrigger} from 'app/components/popups';
```

Comment:
Looks like this is implementing a change suggested in a comment thread around adding a `pages` subdir. Please update doc to include that so this is made clear to all reviewers

Reply:
Done.